### ISSUE TITLE
False Positive on UnusedPrivateMember with interface's default method

### ISSUE THREAD

**Issue by alphaho** [arturbosch/detekt#1613]
Hi,

I've found that when I'm calling a file-private helper method in an interface's default method, detekt would report the helper method is unused(`UnusedPrivateMember`).

For example, with the following code, detekt task would fail:

```kotlin
interface Bar {
    
    fun doSomething() {
        doSomethingElse()
    }
}

private fun doSomethingElse() {

}
```

However, if I change `Bar` from `interface` to `class`, it passes as expected. Which may hint some inconsistency between checking a interface and a class.

## Expected Behavior
It should not report `UnusedPrivateMember` when the file-private method is used in the interface's default method

## Observed Behavior
Detekt task fails and report there's an `UnusedPrivateMember` error:

```
Ruleset: style - 5min debt
        UnusedPrivateMember - [doSomethingElse] at C:\...\xxx.kt:71:1

Overall debt: 5min
```

## Your Environment

* Version of detekt used: 1.0.0-RC14
* Version of Gradle used (if applicable): 5.4
* Operating System and version: Windows 10

**Comment by arturbosch** [comment]
Thanks for reporting! The fix will be included in the next release.

**Comment by alphaho** [comment]
Thanks for the fix!
